gwpy/timeseries/timeseries.py

```diff
@@ -1428,14 +1428,15 @@ def taper(self, side='leftright'):
         out *= planck(out.size, nleft=nleft, nright=nright)
         return out
 
-    def whiten(self, fftlength, overlap=0, method='scipy-welch',
-               window='hanning', detrend='constant', asd=None, **kwargs):
-        """White this `TimeSeries` against its own ASD
+    def whiten(self, fftlength=2, overlap=0, method='scipy-welch',
+               window='hanning', detrend='constant', asd=None,
+               fir=False, filter_duration=1, highpass=0, **kwargs):
```

Comment:
I think I would be more comfortable with `highpass=None` as the default, to make it more obvious that the default is to not apply a FD highpass.

Reply:
Makes sense to me, and that's a pretty easy change, so I'll make that the default.